desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -307,19 +307,24 @@ def check_status(self, notebook, snippet):
 
     return response
 
+
   @query_error_handler
   def progress(self, notebook, snippet, logs=''):
+    progress = 50
     if self.options['url'].startswith('presto://'):
       guid = snippet['result']['handle']['guid']
       handle = CONNECTIONS.get(guid)
-      if not handle:
```

Comment:
Maybe this is also necessary, i am not sure.

Reply:
Adding back!

Reply:
When stats = None, now will throw exception? how about change back use return

Reply:
If stats is None it should return 100 IIUC (line #324)

Reply:
But 325 line, will throw none type has no gets function

Reply:
Ha indeed, will fix soon!

Reply:
https://github.com/cloudera/hue/pull/1560